### PR TITLE
2 get topics

### PR DESCRIPTION
**Pull request by AmadReihun**
Completing task 2 
Get /api/topics

### CODE REVIEW

File: app/controllers/ncTopics.controllers.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ const { fetchTopics} = require("../models/ncTopics.models")
 
 exports.getTopics = (req, res, next) => {
   fetchTopics()
-  .then((data) => {
-    res.status(200).send();
+  .then((topics) => {
+    res.status(200).send({ topics });
+    console.log(topics , '<<<<<< in controller :D ');
```
Comment (nc-christian-dutton) [inline app/controllers/ncTopics.controllers.js:7]: carry on using console log but please remove before pushing!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -12,10 +12,30 @@ beforeEach(() => seed(testData));
...
+            // console.log(body);
+            expect(Object.keys(topics[0]).length).toBe(2);
+            expect(topics).toHaveLength(3);
+            // Checking the data type
+            topics.forEach((topic) => {
+              expect(typeof topic.slug).toBe("string");
+              expect(typeof topic.description).toBe("string");
+            });
+          });
+      });
```
Comment (nc-christian-dutton) [inline __tests__/app.test.js:33]: Good test well done 👍

File: app/errorHandlers.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+exports.handle400s = (err, req, res, next) => {
+  console.log(err);
+  if (err.code === "22P02" || err.code === "23502") {
+    res.status(400).send({ msg: "Bad request" });
+  } else if (err.code === "23503") {
+    res.status(404).send({ msg: "Not found" });
+  } else if (err.status === 400) {
+    res.status(err.status).send({ msg: err.msg });
+  }
+};
```
Comment (nc-christian-dutton) [inline app/errorHandlers.js:10]: You shouldn't push code that doesn't affect the ticket you are on - you have no errors yet.

File: app/controllers/ncTopics.controllers.js (hunk 1 above)
Comment (AmadReihun) [reply app/controllers/ncTopics.controllers.js:7]: Thanks, sure will delete them :)

File: app/errorHandlers.js (hunk 3 above)
Comment (AmadReihun) [reply app/errorHandlers.js:10]: You are totally right, will keep them out of the code for now :)

**Pull request merged**